### ISSUE TITLE
Progress bar disappeared for a while after changing settings

### ISSUE THREAD

**Issue by blackarcher21** [martinrotter/rssguard#73]
#### Brief description of the issue.

Progress bar disappeared  for a while after I changed the settings. I can't tell if the progress also stopped for a while or not. log attached. RSS Guard 3.3.5 nowebengine, Windows 8.1 64-bit

#### How to reproduce the bug?

1. Update feed/category
2. Change some settings
3. Click Apply

#### What is the expected result?

Progress bar keep showing the progress

#### What actually happened?

Progress bar disappeared for a while

#### Other information (logs, see Wiki)

[log.txt](https://github.com/martinrotter/rssguard/files/573872/log.txt)

**Comment by martinrotter** [comment]
So progress bar appeared again after a while? When settings are saved, progress bar is not touched at all. This can be some glitch inside Qt, on my PC, progress just flicks a bit when I open settings dialog, but is still more-or-less visible.

**Comment by martinrotter** [comment]
Progress should not stop, this is for sure only UI problem. Video would be nice, so that I can see the situation and judge if this is our issue or not.

**Comment by blackarcher21** [comment]
yes
https://transfer.sh/U7in6/2016-11-09-22-36-17.mp4

**Comment by martinrotter** [comment]
Thank you for video, I identified the exact problem. When you edit some settings in GUI section, all toolbars (including bottom status bar) are reloaded, thus progress bar may disappear for small amount of time, it will appear again when NEXT feed gets updated after closing settings dialog.

So this is really just cosmetic problem. There needs to be 100% working way of reloading status bar even if feed updates are running at the same moment, my approach works 100%, but I tried to even enhance it to minimize the flickering of progress bar.

That should fix your issue.
